lib/matplotlib/axes/_axes.py

```diff
@@ -4537,11 +4559,8 @@ def arrow(self, x, y, dx, dy, **kwargs):
             ax.annotate("", xy=(0.5, 0.5), xytext=(0, 0),
                 arrowprops=dict(arrowstyle="->"))
 
-        Optional kwargs control the arrow construction and properties:
-
-        %(FancyArrow)s
-
-        **Example:**
+        Examples
+        --------
 
         .. plot:: mpl_examples/pylab_examples/arrow_demo.py
```

Comment:
Can you please add a blank line after this? I am not entirely sure why, but python3.5 is unhappy…